### PR TITLE
[16.0][IMP] account_reconcile_oca: add due date field in account_move_line tree view

### PR DESCRIPTION
**Pull request by ghost**
Shows `due_date` from related `move_id` in `account_move_line` tree view

### CODE REVIEW

File: account_reconcile_oca/models/account_move_line.py [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 
 class AccountMoveLine(models.Model):
 
     _inherit = "account.move.line"
 
+    invoice_due_date = fields.Date(
+        related="move_id.invoice_date_due",
+        string="Invoice Due Date",
+        readonly=True,
+        store=True,
```
Comment (etobella) [inline account_reconcile_oca/models/account_move_line.py]: You don't need to make it stored

**Review by aritzolea — APPROVED** [review]

File: account_reconcile_oca/models/account_move_line.py (hunk 1 above)
Comment (etobella) [reply account_reconcile_oca/models/account_move_line.py]: @ACheung-FactorLibre can you make it not readable please?

Comment (ghost) [reply account_reconcile_oca/models/account_move_line.py]: Done!

File: account_reconcile_oca/models/account_move_line.py [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 from odoo.exceptions import ValidationError
 
 
 class AccountMoveLine(models.Model):
 
     _inherit = "account.move.line"
 
+    invoice_due_date = fields.Date(
+        related="move_id.invoice_date_due",
+        string="Invoice Due Date",
```
Comment (etobella) [inline account_reconcile_oca/models/account_move_line.py]: The string is unnecesary

File: account_reconcile_oca/models/account_move_line.py [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+        readonly=True,
+        copy=False,
```
Comment (etobella) [inline account_reconcile_oca/models/account_move_line.py]: Copy has no sense here

File: account_reconcile_oca/i18n/es.po [hunk 4]
```diff
@@ -505,7 +510,7 @@ msgstr "Marcar como revisado"
 #. module: account_reconcile_oca
 #: model:ir.model.fields,help:account_reconcile_oca.field_account_bank_statement_line__foreign_currency_id
 msgid "The optional other currency if it is a multi-currency entry."
-msgstr "La otra divisa opcional si se trata de una entrada multidivisa."
+msgstr "La moneda alternativa si se trata de un asiento multi-moneda."
```
Comment (etobella) [inline account_reconcile_oca/i18n/es.po]: Translation is never done directly in commits. Use translation.odoo-community.org

**Review by etobella — CHANGES_REQUESTED** [review]

File: account_reconcile_oca/models/account_move_line.py (hunk 3 above)
Comment (ghost) [reply account_reconcile_oca/models/account_move_line.py]: Done

File: account_reconcile_oca/i18n/es.po (hunk 4 above)
Comment (ghost) [reply account_reconcile_oca/i18n/es.po]: Done

File: account_reconcile_oca/models/account_move_line.py (hunk 2 above)
Comment (ghost) [reply account_reconcile_oca/models/account_move_line.py]: Done

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
@ACheung-FactorLibre is it ready for review?

**Comment by ghost** [comment]
> @ACheung-FactorLibre is it ready for review?

Yes

**Pull request merged**